internal/services/media/registration.go

```diff
@@ -67,7 +67,10 @@ func (r Registration) DataSources() []sdk.DataSource {
 }
 
 func (r Registration) Resources() []sdk.Resource {
-	return []sdk.Resource{
-		AccountFilterResource{},
+	if !features.FourPointOhBeta() {
```

Comment:
now that 4.0 has been released we can remove this completely without using the flag

Reply:
I've checked some other resources and noticed they were still behind the flag (e.g. iottimeseriesinsights). I'd like to follow the current behavior, and expect a commit later to do a bulk delete. WDYT?

Reply:
the other `if` statement should be fine to remain because it was put in place before the 4.0 release

Reply:
@catriona-m Done